packages/export/src/watson/index.ts

```diff
@@ -23,49 +20,113 @@ export class WatsonExporter extends Exporter {
     [Botmock.SystemEntity.ORDINAL, Watson.SystemEntity.PERCENTAGE],
     [Botmock.SystemEntity.CURRENCY, Watson.SystemEntity.CURRENCY],
     [Botmock.SystemEntity.DATE, Watson.SystemEntity.DATE],
-    [Botmock.SystemEntity.DURATION, Watson.SystemEntity.TIME],
-    [Botmock.SystemEntity.COUNTRY, Watson.SystemEntity.LOCATION],
-    [Botmock.SystemEntity.NAME, Watson.SystemEntity.PERSON],
+    [Botmock.SystemEntity.DURATION, Watson.SystemEntity.TIME]
   ]);
-  /**
-   * Finds name of first intent connected to response block
-   * @param id Id of response block
-   * @param blocks {@link Botmock.Message[]}
-   */
-  #findConditionsOfBlockId = (id: string, blocks: Botmock.Message[]): string => {
-    const [intentId] = blocks
-      .flatMap(block => block.next_message_ids.filter(next => next.message_id === id))
-      .filter(next => next.intent.value)
-      .map(next => next.intent.value);
-    return this.#intents.find(intent => intent.id === intentId)?.name as string;
+
+  #findNameOfIntent = (id: string) => {
```

Comment:
Let's rename this to: `getIntentName`
```suggestion
  #getIntentName = (id: string) => {
```